lectia-2.js

```diff
@@ -1,3 +1,17 @@
 // https://miorita.scoalaweb.ro/lessons/2-if
 // Scrie un program care să o ajute pe Miorița să ajungă la oricare din pășuni
 // Programul trebuie să funcționeze și cu gard, și fără gard
+reset();
+if (!canMove()) {
+  turnLeft();
+  move();
+  turnRight();
+  move();
+  move();
+  move();
+}
+if (canMove()) {
```

Comment:
Funcționează, dar e puțin tras de păr. Ai fi putut pune `else`, sau și mai bine, să nu pui nici `else`, nici al doilea `if`